pythonzm_Ops_task_views_41.py

```
@@ -0,0 +1,412 @@
+        p_name = request.POST.get('p_name')
+        is_parent = request.POST.get('isParent')
+        new_name = request.POST.get('new_name')
+        path = os.path.join(p_name, name)
+        new_path = os.path.join(p_name, new_name)
+        try:
+            if not os.path.exists(path) and not os.path.exists(new_path):
+                if is_parent == 'true':
+                    os.makedirs(new_path)
+                else:
+                    open(new_path, 'w').close()
+                return JsonResponse({'code': 200, 'msg': '添加成功！'})
```

Ruff (0.13.1)

String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)